kong/db/migrations/core/000_base.lua

```diff
@@ -152,7 +152,7 @@ return {
         "hash_on_cookie"        TEXT,
         "hash_on_cookie_path"   TEXT,
         "slots"                 INTEGER                      NOT NULL,
-        "healthchecks"          JSON
+        "healthchecks"          JSONB
```

Comment:
This migration should truly reflect the state of the database as of 0.14, and we should not diverge away from it. The `14_to_15` migration will still run for freshly bootstrapped databases as well, so there is no need for this change anyway?

Reply:
Thanks for for this. I fully 100% agree.